social_core/backends/oauth.py

```diff
@@ -46,7 +46,6 @@ class OAuthAuth(BaseAuth):
 
     AUTHORIZATION_URL = ""
     ACCESS_TOKEN_URL = ""
-    ACCESS_TOKEN_METHOD = "GET"
```

Comment:
The `BaseAuth` doesn't appear to have an `ACCESS_TOKEN_METHOD` attribute so unless I am missing something this may end-up being undefined in some cases and lead to errors.

Instead of deleting it, why not explicitly set the value to `POST` ? (added bonus it will be visible to anyone looking).

Reply:
Ahhh, tests failed with exactly this error 
```
AttributeError: 'GithubOAuth2' object has no attribute 'ACCESS_TOKEN_METHOD'
```

Reply:
Yes, I reverted this edit after mass removing these from the other backends, but forgot to commit that part :facepalm:.